joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/MacroHandler.scala

```diff
@@ -161,6 +151,25 @@ trait MacroHandler {
       .toBuffer
   }
 
+  /**
+    * Create a full name field that encodes line information that can be picked
+    * up by the MethodStubCreator in order to create a METHOD node with the
+    * correct location information.
+    * */
+  private def fullName(macroDef: IASTPreprocessorMacroDefinition, argAsts: List[Ast]) = {
+    val name = macroDef.getName.toString
+    val fileLocation = macroDef.getFileLocation
+
+    if (fileLocation != null) {
+      val lineNo = fileLocation.getStartingLineNumber
+      val lineNoEnd = lineEnd(macroDef).getOrElse("-1")
+      val fileName = fileLocation.getFileName
+      fileName + ":" + lineNo + ":" + lineNoEnd + ":" + name + ":" + argAsts.size
+    } else {
+      "<empty>:-1:" + name + ":" + argAsts.size
```

Comment:
Very minor nit, but in the failure case we're reporting one line number field, while in the success case we're reporting 2. If we (or anyone) wants to extract this information programmatically, it would be easier if the fields are uniform.

```suggestion
      "<empty>:-1:-1:" + name + ":" + argAsts.size
```

Reply:
true, good catch, will fix